packages/core/src/App/Containers/Modals/exit-traders-hub-modal/exit-traders-hub-modal.jsx

```diff
@@ -40,14 +42,19 @@ const ExitTradersHubModal = ({
     };
 
     const onClickExitButton = async () => {
+        const cr_account = active_accounts.some(acc => acc.landing_company_shortcode === 'svg');
         toggleExitTradersHubModal();
-        setIsPreAppStore(false);
         if (content_flag === ContentFlag.LOW_RISK_CR_EU) {
+            if (!cr_account) {
+                await switchAccount(account_list.find(acc => acc.loginid.startsWith('VRTC'))?.loginid);
+            }
             //if eu is currently selected , switch to non-eu on exiting tradershub
-            await switchToCRAccount();
+            await switchAccount(account_list.find(acc => acc.loginid.startsWith('CR'))?.loginid);
```

Comment:
After switching, shouldnt we set selected region to NON-EU?

Reply:
If i use switchToCRAccount.  After exiting and returning back to traders hub it goes to the Non-EU when its supposed to go to demo

Reply:
Alright.Thanks @thisyahlen-deriv